.changeset/olive-onions-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+"@slashid/react": patch
+---
+
+Improve the OTP flow states
```

Comment:
This message will end up in the automatically generated changelog - we need to be a bit more specific than this. Can we list the changes we did here?

Reply:
Sorry you're right, I will add more details